ts/octorelay.ts

```diff
@@ -174,8 +175,12 @@ $(() => {
           const closeBtn = $(`#navbar_plugin_octorelay #pop-closer-${key}`);
           const cancelBtn = $(`#navbar_plugin_octorelay #cancel-btn-${key}`);
           const timeTag = $(`#navbar_plugin_octorelay #time-tag-${key}`);
-          setCountdown(timeTag, value.upcoming.deadline);
+          const countdownDisposer = setCountdown(
+            timeTag,
+            value.upcoming.deadline
+          );
           const closePopover = () => {
+            countdownDisposer();
             closeBtn.off("click");
             relayBtn.popover("hide");
           };
```

Comment:
I think it's important to dispose early (even if the countdown has it's only self-disposer when not visible),
because there could be a case when the popover closes and appears again for ANOTHER task.
In that case it should have completely another thread of workflow and the old countdown should not interfere.